homeassistant/components/fully_kiosk/services.py

```diff
@@ -52,6 +52,8 @@ async def execute_service(
                         continue
                     if entry.domain != DOMAIN:
                         continue
+                    if DOMAIN not in hass.data or key not in hass.data[DOMAIN]:
```

Comment:
I think we should check the state of the config entry instead. It should be `ConfigEntryState.LOADED`. We can combine all the checks with `or` as the result should be the same.